providers/fips/fipsprov.c

```diff
@@ -1072,6 +1076,11 @@ int ERR_pop_to_mark(void)
     return c_pop_error_to_mark(NULL);
 }
 
+int ERR_count_to_mark(void)
+{
+    return c_count_to_mark(NULL);
```

Comment:
I think this will break backwards compatibility with old libcrypto. Something like:
```suggestion
    return c_count_to_mark != NULL ? c_count_to_mark(NULL) : 0;
```
should work.

Of course it means with old libcrypto this will always return 0 but for this particular function it does not matter that much, IMO.

Reply:
Yes, good point. Will fix (also in legacyprov.c of course).